src/main/groovy/edu/oregonstate/mist/invencycle/core/ErrorPOJO.groovy

```diff
@@ -0,0 +1,9 @@
+package edu.oregonstate.mist.invencycle.core
+
+/**
+ * Error class
+ */
+class ErrorPOJO {
+    String errorMessage
+    Integer errorCode
+}
```

Comment:
You don't need to define the Error class. You should be able to use the one that is in the "api" package: https://github.com/osu-mist/invencycle-api/blob/feature/ECSPCS-219-get-bikes/id/src/main/groovy/edu/oregonstate/mist/api/Error.groovy